package.json

```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "react-tooltip": "^4.2.21",
```

Comment:
[PrimeReact](https://primefaces.org/primereact/showcase/#/tooltip) already provides a Tooltip-Function, so we dont need an additional package for this.

Reply:
Yes will change from react-tooltip to primeReact tooltip